lib/src/extractLevels.js

```diff
@@ -0,0 +1,191 @@
+import fs from 'fs'
+import IfcModel from './IfcModel.js'
+
+
+const IFCSpacesELementsArrAll = []
```

Comment:
How about IfcSpaceElements and move out of the global context.

Reply:
I deleted ifcSpaceElements for now as we won't need it.